tests/unit/engines/summary/test_prompt_summary_engine.py

```diff
@@ -9,7 +9,7 @@
 class TestPromptSummaryEngine:
```

Comment:
Would be nice if there was at least one test that used the context in the template too (currently added tests seem to just supply the context init param, but never test that it effects the output).